js/build-env-config.js

```diff
@@ -13,6 +13,7 @@ if (environment !== `PRODUCTION`) {
 }
 
 var config = {
+  ORIGIN: habitat.get(`ORIGIN`),
```

Comment:
Would adding a note to README to inform devs to create a `.env` file basd on `defaults.env` help? I'm worried if setting defaults with`habitat` plus loading potentially 2`.env` files might be a bit redundant?